kernel/src/thread/task.rs

```diff
@@ -74,8 +74,10 @@ pub fn create_new_user_task(
         };
 
         while !current_thread.is_exited() {
+            ctx.thread_local.fpu().activate();
             // Execute the user code
             let return_reason = user_mode.execute(has_kernel_event_fn);
+            ctx.thread_local.fpu().deactivate();
```

Comment:
How about making the comment describe the whole code block?
```rust
            // Execute the user code
            ctx.thread_local.fpu().activate();
            let return_reason = user_mode.execute(has_kernel_event_fn);
            ctx.thread_local.fpu().deactivate();
```

Describing just the second statement is also correct, however, it's a bit ugly and inconsistent with the comments below.